docs/use/dapp-staking/for-stakers/index.md

```diff
@@ -0,0 +1,111 @@
+import Figure from "/src/components/figure"
+
+# For Stakers
+
+## Overview
+
+Do you love to stake? Or do you want to support your favorite project on Astar Network? 
+
+Discover [Astar dApp Staking](/docs/learn/build2earn.md), a unique mechanism that allows communities to support their favorite teams by staking ASTR or SDN with them. When staking ASTR on a dApp, users not only support the development of innovative apps but also receive staking rewards from inflation.
```

Comment:
This link will not exist after PR https://github.com/AstarNetwork/astar-docs/pull/573 gets merged so please review that PR first, so it gets merged and then use the cross-link to /docs/learn/dapp-staking here